lib/tracker/mode_s/decode.go

```diff
@@ -494,9 +494,20 @@ func decodeFlightNumber(b []byte) []byte {
 	callsign[6] = aisCharset[((b[4]&15)<<2)|(b[5]>>6)]
 	callsign[7] = aisCharset[b[5]&63]
 
+	score := 0
+	for i := 0; i < 8; i++ {
+		if (callsign[i] > 'A' && callsign[i] < 'Z') || callsign[i] == '-' || callsign[i] == ' ' || (callsign[i] > '0' && callsign[i] < '9') {
```

Comment:
Are you excluding A,Z,0 and 9 here? Should it be:

```go
if (callsign[i] >= 'A' && callsign[i] <= 'Z') || callsign[i] == '-' || callsign[i] == ' ' || (callsign[i] >= '0' && callsign[i] <= '9') {
    score++
}
```

Reply:
ahh yeah, I am a doofus